### DISCUSSION TITLE
Help me to find transaction

### DISCUSSION THREAD

**Discussion by lyalkov** [aptos-labs/aptos-developer-discussions#548] [Questions]
### Describe your question in detail.

I send Apt from tangem to bybit, but it didn't come. When I want to check here explorer.aptoslabs.com transactions from my tangem wallet, page with my address won't download, just white screen. How can I find it? Thanks

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

N/A

**Comment by 0xmaayan** [comment]
Hi @lyalkov , what is your wallet address ?

**Reply by lyalkov** [reply]
0x4b9da755bccc9fa1d7aafc238118a420684db60cf8b2be0c93f6e4243660efe6 from this wallet I send token

**Reply by 0xmaayan** [reply]
Thanks! Looks like there is a bug in the Explorer site, but you can use `aptoscan` to see the transactions
https://aptoscan.com/account/0x4b9da755bccc9fa1d7aafc238118a420684db60cf8b2be0c93f6e4243660efe6

**Reply by lyalkov** [reply]
Thanks. I find it https://aptoscan.com/transaction/2009412126
I send it on Eth wallet. Possible to return it or to do something with it? Big amount for me :(

**Reply by 0xmaayan** [reply]
The coins are now live under this account https://aptoscan.com/account/0x00000000000000000000000093404464d55d6f75d8a0ebc13294c39819e9bc85 which is a random account that probably no one has access to.

**Reply by lyalkov** [reply]
thanks
